features-support/hooks/ui-automation.js

```diff
@@ -90,30 +91,41 @@ module.exports = function seleniumHooks() {
         .withCapabilities(capabilities)
         .build();
 
-      // Manage timeouts.
-      timeoutManager = driver.manage().timeouts();
-      timeoutManager.pageLoadTimeout(pageLoadTimeoutms);
-      timeoutManager.implicitlyWait(implicitlyWaitTimeoutms);
+      driver.getSession()
+          .then(function () {
+            // Manage timeouts.
+            timeoutManager = driver.manage().timeouts();
+            timeoutManager.pageLoadTimeout(pageLoadTimeoutms);
+            timeoutManager.implicitlyWait(implicitlyWaitTimeoutms);
 
+            driver.manage().window().maximize()
+                .then(function () {
+                  callback();
+                });
+
+          });
     } catch (error) {
```

Comment:
I tested it and when webdriver fails to start the browser, tests fail which is good enough. No need for "extra" error handling.